src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -55,72 +55,81 @@ Need to edit a doc? Use this table to decide where to work!
 **Continue reading for instructions on how to edit a doc locally.**
 
 ## 1. Set up your local environment [#set-local]
-
 Running the site locally makes testing and previewing large changes much easier. Here's how to get setup:
 
-1. Install GitHub Desktop
-2. Sign in to GitHub Desktop.
+1. Install GitHub Desktop.
+2. Sign in to GitHub Desktop using your GitHub account.
     - On Macs, click on **GitHub Desktop** in the top left corner of your screen and select **Preferences**. Select the blue **Sign In** button and follow the prompts in the browser window. 
-3. Navigate to the [Docs Site repository](https://github.com/newrelic/docs-website) on GitHub.com. 
-4. Click the green **Code** button and then select **Open with [GitHub Desktop](https://desktop.github.com/)**. 
-5. Choose the location where you want the repo, and this will clone the entire repository to your local machine at the designated path. 
+3. Navigate to the [Docs Site repository](https://github.com/newrelic/docs-website). 
+4. Click the green **Code** button and select **Open with [GitHub Desktop](https://desktop.github.com/)**. 
+5. Choose the location where you want the repo. The default location for this folder is `~/Documents/github`.
+6. Clone the entire repository to your local machine. You will not need to clone the repo again.
  
-You can ensure the repo was cloned by navigating to your local GitHub folder (the default is `~/Documents/github`). Once you have cloned the repo, you don't need to clone it again in the future.  
+Check your repo was cloned by navigating to your local GitHub folder. 
+
+## 2. Create a branch and edit the docs [#edit-doc]
+In GitHub Desktop, you will see three tabs labeled **Current Repository**, **Current Branch**, and **Fetch origin**. The default branch is **Develop**, so it's time to create a branch.
+
+1. Click on **Current Branch** and select **New Branch**. Give it a meaningful name.
+2. Navigate to a doc you want to edit.
+    - Use Finder or File Explorer with the file path. For example, if you wanted to edit a Python agent doc, you would navigate to:  `~/Documents/github/docs-website/src/content/docs/agents/python-agent/hosting-services/python-agent-stackato.mdx`
+3. [Edit the doc](https://docs.newrelic.com/docs/style-guide/writing-docs/processes-procedures/create-edit-content) in your text editor of choice. You should write docs with [markdown language](https://www.markdownguide.org/basic-syntax).
```

Comment:
We use VS Code. I think it's ok to say that.

Reply:
My only concern is consistency. We haven't mentioned VS Code consistently so far, and only sporadically from what I have seen. Do you think it's best to introduce VS Code in one sweep, or should I go over the doc and add VS Code to the steps now?